src/main/scala/com.snowplowanalytics.snowplow.micro/ConfigHelper.scala

```diff
@@ -49,9 +53,28 @@ private[micro] object ConfigHelper {
 
   implicit val sinkConfigHint = new FieldCoproductHint[SinkConfig]("enabled")
 
+  implicit val clockProvider: Clock[Id] = new Clock[Id] {
+    final def realTime(unit: TimeUnit): Id[Long] =
+      unit.convert(System.currentTimeMillis(), TimeUnit.MILLISECONDS)
+
+    final def monotonic(unit: TimeUnit): Id[Long] =
+      unit.convert(System.nanoTime(), TimeUnit.NANOSECONDS)
+  }
+
+  type EitherS[A] = Either[String, A]
+
```

Comment:
Copied from Enrich

Reply:
Perhaps it's worth putting in your two comments on github as code comments?